blog-service/2023-02-07-slo-log-search.md

```diff
@@ -0,0 +1,18 @@
+---
+title: SLO Dashboard Panel Log Search
+image: https://www.sumologic.com/img/logo.svg
+keywords:
+  - log search
+  - slo
+  - dashboards
+hide_table_of_contents: true
+authors:
+  - url: https://help.sumologic.com/release-notes-service/rss.xml
+    image_url: /img/release-notes/rss-orange.png
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+You can now launch a Log Search session directly from an SLO dashboard panel, giving you the ability to drill down further into your SLO data. [Learn more](https://help.sumologic.com/docs/observability/reliability-management-slo/dashboards).
```

Comment:
We can add - `You can further add the log search results into any other dashboard, allowing you to correlate your SLO performance with other data`.